lib/moon_web/pages/components/typography_page.ex

```diff
@@ -11,7 +11,7 @@ defmodule MoonWeb.Pages.Components.TypographyPage do
     default: [
       %{
         to: "#",
-        name: "Typography"
+        name: "Components"
```

Comment:
This is not v1 and not v2?

Reply:
Icons, Flags and Typography remained just "Components" as I wasn't sure if it is needed to define versions for them?